docs/guides/modules/orchestrate/pages/pipeline-variables.adoc

```diff
@@ -195,11 +195,12 @@ Pipeline parameters which are defined in configuration are always in scope, with
 [#pipeline-parameters-and-dynamic-configuration]
 === Pipeline parameters and dynamic configuration
 
-When using setup workflows to pass control of the pipeline to a second configuration, all pipeline parameters that are declared in the setup workflow must also be declared, with the same defaults, in the second configuration.
+All parameters that passed from the setup workflow, via the `continue` job or command from the link:https://circleci.com/developer/orbs/orb/circleci/continuation[continuation orb] using the `parameters` parameter must be declared as pipeline parameters in the continuation configuration.
```

Comment:
Pipeline parameters can be passed from the setup to the continuation workflow using the `continue` job or command from the link:https://circleci.com/developer/orbs/orb/circleci/continuation[continuation orb]. To pass pipeline parameters use the `parameters` parameter from the `continue` job/command. Any pipeline parameters passed in this way must be declared as pipeline parameters in the continuation configuration.